cypress/integration/pages/FIX/tests.js

```diff
@@ -0,0 +1,98 @@
+// Limiting to only one service
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing features that may differ across services but share a common logic e.g. translated strings.
```

Comment:
Feature Index Cypress tests pretty much duplicate the front page cypress tests, but check for a non-hidden title.